seg_lapa/train.py

```diff
@@ -27,20 +29,21 @@ def training_step(self, batch, batch_idx):
         inputs, labels = batch
         outputs = self.model(inputs)
         loss = self.cross_entropy_loss(outputs, labels)
+        batch_loss = loss / len(batch[0])
+        wandb.log({"Train/BatchWise Loss": batch_loss})
 
         # to aggregate epoch metrics use self.log or a metric. self.log logs metrics for each training_step.
         # It also logs the average across the epoch, to the progress bar and logger
         # "train_loss" is a reserved keyword
-        self.log("loss", loss, on_step=True, on_epoch=True, prog_bar=True, logger=True)
-
         return loss
 
     def validation_step(self, batch, batch_idx):
         inputs, labels = batch
         outputs = self.model(inputs)
         loss = self.cross_entropy_loss(outputs, labels)
+        batch_loss = loss/len(batch[0])
```

Comment:
Let's rename it to avg_batch_loss, just to reduce confusion against loss vs batch_loss
```suggestion
        avg_batch_loss = loss/len(batch[0])
```